### PR TITLE
Update Depreciated Pillow Methods

### PR DESCRIPTION
**Pull request by vertyco**
Using latest pillow antialias method `Resampling.LANCZOS`
Using latest pillow getsize method `getlength`

### CODE REVIEW

File: pilmoji/core.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ def text(
             The rescaling factor for emojis. This can be used for fine adjustments.
             Defaults to the factor given in the class constructor, or `1`.
         emoji_position_offset: Tuple[int, int]
-            The emoji position offset for emojis. The can be used for fine adjustments.
+            The emoji position offset for emojis. Then can be used for fine adjustments.
```
Comment (jay3332) [inline pilmoji/core.py:255]: This actually should've been "This" instead of "The" or "Then"

File: pilmoji/core.py [hunk 2]
```diff
@@ -309,7 +309,7 @@ def text(
                 with Image.open(stream).convert('RGBA') as asset:
                     width = int(emoji_scale_factor * font.size)
                     size = width, math.ceil(asset.height / asset.width * width)
-                    asset = asset.resize(size, Image.ANTIALIAS)
+                    asset = asset.resize(size, Image.Resampling.LANCZOS)
```
Comment (jay3332) [inline pilmoji/core.py]: `Image.LANCZOS` is documented so it should be used over `Image.Resampling.LANCZOS`

**Review by jay3332 — CHANGES_REQUESTED** [review]
LGTM, however `getlength` might not be compatible with people on older versions of pilmoji so I don't know if I should use it.

File: pilmoji/core.py (hunk 2 above)
Comment (vertyco) [reply pilmoji/core.py]: changed it to Image.LANCZOS

**Comment by vertyco** [comment]
I've implemented this if you think it will suffice, since getsize was depreciated in 9.2
```             
                if PIL.__version__ >= "9.2.0":
                    width = font.getlength(content)
                else:
                    width, _ = font.getsize(content)
```

File: pilmoji/core.py (hunk 2 above)
Comment (vertyco) [reply pilmoji/core.py]: Oddly i am using Resampling in one of my projects and it also works, not sure if there is a difference other than not being typehinted

**Comment by vertyco** [comment]
making all the necessary adjustments and reopening this

**Pull request closed without merging**